### ISSUE TITLE
Add md5 checksum files for the binaries

### ISSUE THREAD

**Issue by ppanyukov** [ppanyukov/git-build#1]
So we can verify downloads.

**Comment by bcc** [comment]
MD5? is this 1997? ;)

**Comment by ppanyukov** [comment]
@bcc What's wrong with md5 for verifying that we downloaded everything correctly?

Can have `sha<whatever>` if this is the more general preference these days.

**Comment by bcc** [comment]
Yes, use sha256. MD5 is broken and shouldn't be used for anything these days, as it's relatively easy to generate collisions. Or better still GPG sign :)
